modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -80,6 +80,13 @@ After the success message you can verify the newly added channels in the {webui}
 
 To monitor the syncing progress of each channel, check the log files in the [path]``/var/log/rhn/reposync`` directory on the {productname} Server.
 
+[IMPORTANT]
+====
+If a product is provided by both the {payg} instance and by the {scc} subscription, it will appear only once in the products list.
```

Comment:
```suggestion
If a product is provided by both the {payg} instance and one of the {scc} subscriptions, it will appear only once in the products list.
```